src/prep_for_wbm.R

```diff
@@ -0,0 +1,66 @@
+# Prepare data ahead of inputting into the NPS WBM. Includes grabbing necessary geospatial data for each location of interest.
+
+prep_for_wbm <- function(climate_data = clim_all,
+                         name = "site_no",
+                         Snowpack.Init = 0,
```

Comment:
We'll want to define Snowpack.Init, Soil.Init, and Shade.Coeff in the run_wbm function in case we decide to change or calibrate those when we run the model. Suggest removing and adding to run_nps_wbm (note, they're also listed as function args in that though they don't actually change within the model run).  See suggested change below.